### ISSUE TITLE
Focus mode css snippet for opacity not working

### ISSUE THREAD

**Issue by ronrdev** [ryanpcmcquen/obsidian-focus-mode#39]
Here is the snippet I have installed. It has no effect.

```
.focus-mode .cm-s-obsidian div:not(.CodeMirror-activeline) > .CodeMirror-line {
    opacity: 40% !important;
    filter: saturate(1) !important;
}

```

**Comment by ryanpcmcquen** [comment]
Not sure what you are trying to accomplish, but the `opacity` and `filter` values should match. Although it doesn't seem like `activeline` gets added to the DOM anymore in the latest version of Obsidian.

**Comment by ryanpcmcquen** [comment]
Asking about active line support here:
https://forum.obsidian.md/t/no-more-codemirror-activeline-class/46591

**Comment by ronrdev** [comment]
The typewriter scroll plugin has a zen mode that lowers the opacity of all paragraphs except where the cursor is located. This only effects one pane. I don't know how this is achieved but maybe could look under the hood at that. It's working. To test, installed typewriter scroll plugin, opened command window, typed zen to find and turn on zen mode.

https://github.com/deathau/cm-typewriter-scroll-obsidian

![Screenshot_2022-10-30_19-41-55](https://user-images.githubusercontent.com/13022919/198907781-5a507986-07a4-498b-ac38-aa83a6c6fddf.png)

**Comment by ryanpcmcquen** [comment]
Does it work in Obsidian 1.0.3?

**Comment by ronrdev** [comment]
Yes, I just took that screenshot before posting.

**Comment by ryanpcmcquen** [comment]
What other plugins do you have installed?

**Comment by ronrdev** [comment]
Yes- updated the snippet. Also tried changing 1 to 40%, with the idea to lower opacity on non active lines.
```
.focus-mode .cm-s-obsidian div:not(.CodeMirror-activeline) > .CodeMirror-line {
    opacity: 1 !important;
    filter: saturate(1) !important;
}
```
I am checking in the sandbox vault, one plugin at a time.
typewriter scroll zen mode works
focus mode plugin does not

**Comment by ryanpcmcquen** [comment]
Should be:
```css
.focus-mode .cm-s-obsidian div:not(.CodeMirror-activeline) > .CodeMirror-line {
    opacity: 0.4 !important;
    filter: saturate(0.4) !important;
}
```

**Comment by ronrdev** [comment]
That still has no effect, but thanks for the syntax for setting opacity correctly.
Looking at the inspector the active line is labeled like this.

![Screenshot_2022-10-30_23-12-33](https://user-images.githubusercontent.com/13022919/198923458-1e332294-7266-4ad2-93c7-abb5df72e9a8.png)

And I know the codemirror cursor changed to cm-cursor.

**Comment by ronrdev** [comment]
I got the plugin to do the opposite. It's making my active line faded, but it's having an effect.

```
.focus-mode .cm-s-obsidian div:not(.cm-line) > .cm-active {
    opacity: 0.4 !important;
    filter: saturate(0.4) !important;
}
```

**Comment by ronrdev** [comment]
I got it. 

```
.focus-mode .cm-s-obsidian .cm-line:not(.cm-active) {
    opacity: 0.4 !important;
    filter: saturate(0.4) !important;
}
```

**Comment by ryanpcmcquen** [comment]
Nice work!
Will you try the latest version and let me know if you have any issues?
https://github.com/ryanpcmcquen/obsidian-focus-mode/releases/tag/1.11.0

**Comment by ronrdev** [comment]
I can confirm in the latest version this is working as expected.
